src/components/AppRouter.jsx

```diff
@@ -0,0 +1,22 @@
+import { Route, Routes } from 'react-router-dom';
+import Home from '../pages/Home';
+import ProductList from '../pages/ProductList';
+import ProductDetail from '../pages/ProductDetail';
+import SearchResults from '../pages/SearchResults';
+import ShoppingCart from '../pages/ShoppingCart';
+import Checkout from '../pages/Checkout';
+
+const AppRouter = () => {
+  return (
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="product-list" element={<ProductList />} />
+      <Route path="product-detail" element={<ProductDetail />} />
+      <Route path="search-results" element={<SearchResults />} />
+      <Route path="shopping-cart" element={<ShoppingCart />} />
+      <Route path="checkout" element={<Checkout />} />
+    </Routes>
+  );
+};
```

Comment:
As we talked this is out of scope for this deliverable but I'm glad you implemented it from the beginning